python/dask_cudf/dask_cudf/backends.py

```diff
@@ -473,6 +475,26 @@ def sizeof_cudf_series_index(obj):
     return obj.memory_usage()
 
 
+@normalize_token.register(cudf.core.frame.Frame)
+def _normalize_cudf_frame(obj):
+    # Only move data to pandas if we "need" to
+    if config.get("tokenize.ensure-deterministic", False):
```

Comment:
ensure-deterministic is not the default in dask, right?

Reply:
Right, this is `False` by default in `dask.dask`, but [is now `True` in dask-expr](https://github.com/dask-contrib/dask-expr/pull/223). I'm slightly hesitant to add a new D->H transfer to dask-cudf workflows having `from_pandas`/`cudf`. Although I doubt the overhead will be significant, I saw this config as a pathway to punt on that decision for now.